contrib/snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: monero
-version: 0.10.2-1
+version: 0.11.1-1
```

Comment:
If this is meant to follow the monero versioning, then this should probably be 0.11.1.0-1